integration-tests/tests/confirmDownloadPage.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { AxiosResponse } from 'axios'
+import { sendRequest } from './utils/request/sendRequest'
+import { parse } from 'node-html-parser'
+import crypto from 'crypto'
+import { TriggerEndOfFlowSQSPayload } from './utils/types/sqsPayload'
+import { getIntegrationTestEnvironmentVariable } from './utils/getIntegrationTestEnvironmentVariable'
+import { invokeSQSOperationsLambda } from './utils/aws/invokeSQSOperationsLambdaFunction'
+import { pollNotifyMockForDownloadUrl } from './utils/notify/pollNotifyMockForDownloadUrl'
+
+describe('Confirm download page', () => {
+  let randomId = ''
+  let fileContents = ''
+
+  beforeEach(async () => {
+    randomId = crypto.randomUUID()
+    fileContents = crypto.randomUUID()
```

Comment:
Why do the file contents have to be a different id? Can they not be the same id?

Reply:
Done.